packages/@angular/cli/utilities/stats.ts

```diff
@@ -41,7 +39,18 @@ export function statsToString(json: any, statsConfig: any) {
         .join('');
 
       return `chunk {${y(chunk.id)}} ${g(files)}${names}${size} ${initial}${flags}`;
-    }).join('\n')}
+    }).join('\n');
+
+  const unchangedChunkNumber = json.chunks.filter((chunk: any) => !chunk.rendered).length;
+  const unchangedChunksMessage = unchangedChunkNumber > 0
+    ? `${unchangedChunkNumber} unchanged chunks`
+    : '';
+
+  return rs(stripIndents`
+    Date: ${w(new Date().toISOString())}
+    Hash: ${w(json.hash)}
+    Time: ${w('' + json.time)}ms
+    ${changedChunksStats ? changedChunksStats + '\n' : ''}${unchangedChunksMessage}
```

Comment:
Thanks to this pullrequest. @filipesilva  What you think to reorder console & get little bit more concis (because in dev mode, imo, short is better)
Before: 
![image](https://user-images.githubusercontent.com/2789203/34029944-40a01588-e16b-11e7-8866-613957d4b6d1.png)

After:
![image](https://user-images.githubusercontent.com/2789203/34029934-322e024e-e16b-11e7-9f72-5e854c465710.png)

Reply:
I think this looks really good :)

Reply:
I like the shorter as well.

Reply:
I've updated the first message to show the shorter output in the after snippet.